File: UmbrellaV1/Models/Advertisement.cs
using System;
using System.Collections.Generic;

namespace UmbrellaV1.Models
{
    public partial class Advertisement
    {
        public long AdvertisementId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public long UserId { get; set; }
        public long CityId { get; set; }
        public DateTime CreatedDate { get; set; }
        public long SubCategoryId { get; set; }

        public City City { get; set; }
        public SubCategory SubCategory { get; set; }
        public User User { get; set; }
    }
}

File: UmbrellaV1/ClientApp/src/app/shared/models/advertisement.ts
import { City } from './city';
import { SubCategory } from './sub-category';
import { User } from './user';
import { Moment } from 'moment';
import { Image } from './image';

export class Advertisement {
  public AdvertisementId: number;
  public Title: string;
  public Description: string;
  public Price: number;
  public UserId: number;
  public CityId: number;
  public CreatedDate: Moment;
  public SubCategoryId: number;
  public City: City;
  public SubCategory: SubCategory;
  public User: User;
  public Image: Image[];
  }
